cmd/swapd/main.go

```diff
@@ -353,11 +355,12 @@ func (d *daemon) make(c *cli.Context) error {
 	}
 
 	sm := swap.NewManager()
-	backend, err := newBackend(d.ctx, c, env, cfg, devXMRMaker, devXMRTaker, sm, host)
+	backend, err := newBackend(d.ctx, c, env, cfg, devXMRMaker, devXMRTaker, sm, host, ec)
 	if err != nil {
 		return err
 	}
 	defer backend.Close()
+	log.Infof("created backend with monero endpoint %s and ethereum endpoint %s", backend.Endpoint(), ethEndpoint)
```

Comment:
It might make the code easier to understand if we change the backend to use a `has-a` instead of an `is-a` relationship with the monero wallet at some point.